src/components/DistanceRequest/DistanceRequestFooter.tsx

```diff
@@ -96,7 +96,7 @@ function DistanceRequestFooter({waypoints, transaction, mapboxAccessToken, navig
                         onPress={() => navigateToWaypointEditPage(Object.keys(transaction?.comment?.waypoints ?? {}).length)}
                         text={translate('distance.addStop')}
                         isDisabled={numberOfWaypoints === MAX_WAYPOINTS}
-                        innerStyles={[styles.ph10]}
+                        innerStyles={[styles.pl10, styles.pr10]}
```

Comment:
Unnecessary change?

Reply:
@s77rt please check this one https://github.com/Expensify/App/issues/38266#issuecomment-1996757522

Reply:
Interesting!